jenkins-plugin/src/test/java/org/jenkinsci/plugins/pipeline/maven/AbstractIntegrationTest.java

```diff
@@ -71,12 +81,31 @@ public abstract class AbstractIntegrationTest {
     public static void setupWatcher() {
         buildWatcher = new BuildWatcher();
         runBeforeMethod(buildWatcher);
+
+    }
+
+    public static void initializeH2(Jenkins jenkins) throws Exception {
+        File databaseRootDir = new File("target", "h2-test");
+        String jdbcUrl = "jdbc:h2:file:" + new File(databaseRootDir, "jenkins-jobs").getAbsolutePath() + ";" +
```

Comment:
you can not reliably make a file URL from a File path  (run the test in a directory with a space for example).  the location can contain non URL safe paths - use `File.toURI().toURL()` and then add that to the special prefix and add your postfix.